index.bs

```diff
@@ -215,7 +215,7 @@ spec: webidl; urlPrefix: https://heycam.github.io/webidl/#
         In the context of this specification, a MIME type is also called content
         type. A <dfn>valid media MIME type</dfn> is a string that is a <a>valid
         MIME type</a> per [[mimesniff]] and must have only one parameter that is
```

Comment:
I'm made a mistake in this proposal. When codecs= is supplied (most of the time), we do want just one media codec described. But there are some cases when codecs= is not supplied that are also valid (e.g. "audio/aac", "audio/mp3" and "audio/flac"). 

Potential fix:
... per [[mimesniff]]. For types where the codec is not implied, a codec parameter must be provided containing only one value that describes a single codec. No other parameters are allowed.